### PR TITLE
Excluded tests/schema subtree from source archive

### PR DESCRIPTION
**Pull request by andy-maier**

### CODE REVIEW

File: docs/changes.rst [hunk 1]
```diff
@@ -47,6 +47,10 @@ Released: not yet
 * Added '.pytest_cache' and '.ruff_cache' directories to be removed in
   'make clean'.
 
+* Excluded 'tets/schema' from the source distribution archive. This is where
```
Comment (KSchopmeyer) [inline docs/changes.rst:50]: change tets/schema to tests/schema

**Review by KSchopmeyer — COMMENTED** [review]
Should probably have an entry in the incompatible changes list since it is a surprise if for some reason users were actually using that schema

one other comment

File: docs/changes.rst (hunk 1 above)
Comment (andy-maier) [reply docs/changes.rst:50]: DONE

**Comment by andy-maier** [comment]
On your comment about incompatible changes: I clarified in the change log that it is not an incompatible change, because the source distribution archive on Pypi never had the schema files, only a locally created one had.

**Pull request merged**